themes/default/content/blog/google-cloud-pulumi-import-account-scraper/index.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "Google Cloud: Bulk Importing Resources into Pulumi"
+date: 2023-03-07
+meta_desc: "Learn how to bulk import your Google Cloud resources into Pulumi"
+meta_image: "meta.png"
+authors:
+    - josh-kodroff
+tags:
+    - google-cloud
+    - import
+---
+
+Point and click in the console is great when you're first starting out learning a new cloud or managed service, but it quickly becomes a hindrance when cloud is widely adopted by an organization. The point at which the term "widely adopted" becomes applicable to your situation differs, but at some point in their careers many infrastructure and platform engineers are faced with situation where a large number of critical infrastructure resources were created through "clickops" with no ability to track changes, reproduce environments consistently, and so on. When this happens (and it will probably happen to many of you), it's time to import those resources into infrastructure as code.
```

Comment:
```suggestion
Point and click in the console is great when you're first starting out learning a new cloud or managed service, but it quickly becomes a hindrance when cloud infrastructure is widely adopted by an organization. The point at which the term "widely adopted" becomes applicable to your situation differs, but at some point in their careers, many infrastructure and platform engineers are faced with situations where a large number of critical infrastructure resources were created through "click ops" with no ability to track changes, reproduce environments consistently, and so on. When this happens (and it will probably happen to many of you), it's time to import those resources into infrastructure as code.
```